### PR TITLE
Alexander Gaivanuk - lab1

### PR DESCRIPTION
**Pull request by alexgaiv**

### CODE REVIEW

**Comment by alexgaiv** [comment]
@PovelikinRostislav, @Maxim-Doronin, @allnes, @komarovn, @nik709 ревьюйте, ребзя

**Comment by nik709** [comment]
👍

File: modules/complex-number/test/test_gaivanuk_alexander_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+#include "include/complex_number.h"
+
+TEST(Gaivanuk_Alexander_ComplexNumberTest, Check_default_ctor) {
+    ComplexNumber z(0.0, 0.0);
+
+    EXPECT_EQ(0.0, z.getRe());
+    EXPECT_EQ(0.0, z.getIm());
+}
+
+TEST(Gaivanuk_Alexander_ComplexNumberTest, Can_create_with_im_and_re_parts) {
```
Comment (Maxim-Doronin) [inline modules/complex-number/test/test_gaivanuk_alexander_complex_number.cpp]: This test (and some more) is a copy of the test in test_complex_number.cpp file

**Comment by alexgaiv** [comment]
@Maxim-Doronin, review again pls

File: modules/complex-number/test/test_gaivanuk_alexander_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
+// Copyright 2017 Alexander Gaivanuk
+
+#include <gtest/gtest.h>
+#include <limits>
+
+#include "include/complex_number.h"
+
+const double eps = std::numeric_limits<double>::epsilon();
+
+TEST(Gaivanuk_Alexander_ComplexNumberTest, Check_default_ctor) {
```
Comment (PovelikinRostislav) [inline modules/complex-number/test/test_gaivanuk_alexander_complex_number.cpp]: Looks lika a copy of Can_Create_Zero test in test_korniakov_kirill_complex_number.cpp

**Comment by Lyzzerg** [comment]
updated test 👍

File: modules/complex-number/test/test_gaivanuk_alexander_complex_number.cpp (hunk 2 above)
Comment (alexgaiv) [inline modules/complex-number/test/test_gaivanuk_alexander_complex_number.cpp]: fixed

**Comment by PovelikinRostislav** [comment]
:+1:

**Comment by alexgaiv** [comment]
@kirill-korniakov

**Pull request merged**